submissions/benjaminKamena/Dom/product.js

```diff
@@ -0,0 +1,64 @@
+// product validation
+let quantity = document.getElementById("quantity"),
+    price = document.getElementById("price"),
+    shippingCost = document.getElementById("shippingCost"),
+    total = document.getElementById("totalCost"),
+    submit = document.getElementById("btn"),
+    perUnit = price.innerHTML,
+    productPrice = perUnit * quantity.value;
```

Comment:
The product price isn't calculated. The shipping cost is. @BenjaminKamena

Reply:
@ToluAkin it is calculated. The quantity * perUnitPrice